### PR TITLE
Add bot702 to SteamBar

### PR DESCRIPTION
**Pull request by ExtremTechniker**
Not tested!

On my pc the module `skate-util` is not found, so I can't test it.

### CODE REVIEW

File: stream-bar/extension/StreamBarManager.ts [hunk 1]
```diff
@@ -34,11 +36,27 @@ export class StreamBarManager extends Manager {
...
-
+		this.register(this.twitchClient, "TwitchClient", () => this.initTwitchClient());
         // TODO: Write frontend
-        // TODO: Add bot702 and !song
     }
 
+	async initTwitchClient(): Promise<void> {
+		await this.twitchClient?.getClient()?.join("#skate702");
+		this.twitchClient?.getClient()?.onMessage(async (channel, _, message) => {
+			if (message.startsWith("!song")) {
```
Comment (sebinside) [inline stream-bar/extension/StreamBarManager.ts]: could contain `toLowerCase` to also deal with `!SONG` and `!sOnG`

File: stream-bar/extension/StreamBarManager.ts [hunk 2]
```diff
@@ -34,11 +36,27 @@ export class StreamBarManager extends Manager {
...
+				if (message.split(" ").length === 2) {
+					const target = message.split(" ")[1];
+					await this.retrieveCurrentSong();
+				this.twitchClient?.getClient()?.say(channel, `@${target} Current song is "${this.streamBarInfo.value.songName}" - "${this.streamBarInfo.value.artistName}"`);
```
Comment (sebinside) [inline stream-bar/extension/StreamBarManager.ts]: Just wondering: Are song names not usually displayed the other way around, meaning: `artist - song`?

**Comment by sebinside** [comment]
Had a quick look, seems to be fine. If you still have trouble setting up skates-bundles, we can try to merge it nevertheless and fix it in the next coding stream ;)

**Comment by ExtremTechniker** [comment]
Now I have found the error, why its wasn't workin on my pc. It was simply that I forgot to add `skate-bundles` to the workspaces in `nodecg-io` again, because I pull new changes... 
But now I can test it and everything is working fine. So it could be merged.

**Pull request merged**